tests/testthat/test-generate_virtual_pop.R

```diff
@@ -23,3 +23,9 @@ test_that('Check the distribution', {
   helper_correct_values <- rexp(1000, 5)
   expect_equal(helper_object1, helper_correct_values)
 })
+
+test_that("Input errors no catching", {
```

Comment:
Minor wording suggestion
```suggestion
test_that("Check that input errors are being caught as expected", {
```